samples/python/classification_sample_async/README.md

```diff
@@ -78,12 +78,12 @@ To run the sample, you need specify a model and image:
    omz_downloader --name alexnet
    ```
 
-3. If a model is not in the IR or ONNX format, it must be converted. You can do this using the model converter:
+3. If a model is not in the OpenVINO IR or ONNX format, it must be converted with Model Converter:
```

Comment:
This changes the meaning. You definitely do not need Model Converter to convert a model to IR, this is just one way of doing this. I would recommend leaving this as is.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14168/commits/f15873dd7ee0277c2b5471c4367febc2e75ef7e4